webpack/assets/javascripts/foreman_tools.test.js

```diff
@@ -57,3 +57,51 @@ describe('deprecate', () => {
     expect(console.warn).toHaveBeenCalledWith('DEPRECATION WARNING: you are using deprecated oldtest, it will be removed in Foreman 1.42. Use tfm.tools.newtest instead.');
   });
 });
+
+/* eslint-disable max-statements */
+describe('initTypeAheadSelect', () => {
```

Comment:
I guess what I would want to test here is that there is a field that has this initialized and check that select2 has the correct response, without mocking out everything inside the function itself. So I guess just mocking the ajax calls to get the correct responses, and making sure the field is populated correctly.